### ISSUE TITLE
Double-Dealing

### ISSUE THREAD

**Issue by TheBearJew6** [throneteki/throneteki#2455]
I had an issue with Double-Dealing plot. I used it on my second turn, and on the first turn my opponent had used A Noble Cause, which reduces the cost of the 1st Lord/Lady by 2. I collected 6 for marshaling, and marshaled Victarion Greyjoy 1st, which costs 6 and is a lord; instead of reducing it 2 and taking 4 of my 6, it took my entire 6. Unless I missed something, this is a bug. It should have taken 4 and left me with 2.

![wtf](https://user-images.githubusercontent.com/49090637/55273515-959c4b00-529a-11e9-9b23-1439d57ccf03.JPG)

**Comment by ystros** [comment]
Thanks for the detailed report and screenshot! It's less of a bug and more the fact that Double-Dealing is currently not implemented. Copying the constant abilities and triggered actions of another card is non-trivial, so we've focused on other things.

It will get implemented eventually, but it's likely going to be complicated and require reworking a lot of how things work today.

**Comment by TheBearJew6** [comment]
Fair enough. I won't use it then. I just didn't see it on the list of unimplemented/partially implemented cards, so I thought that I should let you know; you might want to add it to the list, so that others are aware, and so they don't also suffer a fatal misstep in their game when it does not do what it is supposed to do.
The report was not meant as a complaint, but rather to inform you. I understand that you guys work hard to build the game, and I have been waiting for years for something like this- ever since my group of friends who played the physical version of the game went our separate ways. I can imagine that programming it to copy every single possible abilities/actions would take a lot of work, so I completely understand. I am grateful for the game and your work so far, and it seems to work well otherwise; one card not working yet isn't going to stop me from playing. I am especially grateful that it is free-to-play, because I am rather broke right now. Perhaps if I was paying to play, I would be frustrated by it not being implemented yet (but it still available in deck-building), but since we all get to enjoy your work for free, we have no right to complain or be frustrated. Thanks again, keep up the good work, and I will donate as soon as I have some spare cash. Seven blessing upon you.

**Comment by PeteCrighton** [comment]
You are probably aware, since Double-Dealing is not implemented, but currently it shows my own plots’ used pile and not my opponent’s.
